src/main/java/com/backbase/ct/bbfuel/util/ParserUtil.java

```diff
@@ -3,17 +3,26 @@
 import static java.util.Arrays.asList;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.FileInputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.util.List;
+import java.util.Optional;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
 
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class ParserUtil {
 
     private static ObjectMapper mapper = new ObjectMapper();
 
     public static <T> T convertJsonToObject(String jsonLocation, Class<T> valueType) throws IOException {
         InputStream resourceAsStream = valueType.getClassLoader().getResourceAsStream(jsonLocation);
+        if (resourceAsStream == null) {
+            resourceAsStream = new FileInputStream(jsonLocation);
```

Comment:
Do we need to close this inputstream near the end? Or does the mapper take care of that for us

Reply:
no we don't need to, Jackson will [close](https://fasterxml.github.io/jackson-core/javadoc/2.8/com/fasterxml/jackson/core/JsonParser.html#close()). The ObjectMapper instance calls this method here [_readMapAndClose](https://fasterxml.github.io/jackson-databind/javadoc/2.8/com/fasterxml/jackson/databind/ObjectMapper.html#_readMapAndClose())